content/06.quality_criteria.md

```diff
@@ -26,23 +29,173 @@ reducing the likelihood of service disruption.
 * **[QC.Acc03]** Source code produced within the scope of a broader development project **SHOULD**
   reside in a common organization of a version control repository hosting service.
 
-## 4.2. Licensing [QC.Lic]
+### 4.1.2. Code Workflow [QC.Wor]
+
+A change-based approach is accomplished with a branching model.
+
+* **[QC.Wor01]** The main branch in the source code repository **MUST** maintain a working state
+  version of the software component.
+
+  * **[QC.Wor01.1]** Main branch **SHOULD** be protected to disallow force pushing, thus preventing
+    untested and un-reviewed source code from entering the production-ready version.
+
+  * **[QC.Wor01.2]** New features **SHOULD** only be merged in the main branch whenever the SQA
+    criteria is fulfilled.
+
+* **[QC.Wor02]** New changes in the source code **MUST** be placed in individual branches.
+
+  * **[QC.Wor02.1]** It is **RECOMMENDED** to agree on a branch nomenclature, usually by prefixing,
+    to differentiate change types (e.g. feature, release, fix).
+
+* **[QC.Wor03]** The existence of a secondary long-term branch that contains the changes for the
+  next release is **RECOMMENDED**.
+
+  * **[QC.Wor03.1]** Next release changes **SHOULD** come from the individual branches.
+
+  * **[QC.Wor03.2]** Once ready for release, changes in the secondary long-term branch are merged
+    into the main branch and versioned. At that point in time, main and secondary branches are
+    aligned. This step **SHOULD** mark a production release.
+
+### 4.1.3. Code Management [QC.Man]
```

Comment:
the parent section is also called "Code Management".. hard to find alternatives, maybe something like "code handling", "code specifics", ..

Reply:
agree
changed to Code handling